docs/installation.md

```diff
@@ -19,24 +19,18 @@ cd metamist
 
 ### Install python requirements
 
-It is recommended to use [pyenv](https://github.com/pyenv/pyenv) and [virtualenv](https://virtualenv.pypa.io/en/latest/installation.html) to manage the Python version and virtual environment for Metamist. Since Metamist currently is deployed with python 3.11, you should also use 3.11 locally to ensure that any changes you make are compatible with the deployed version of Metamist.
+Make sure [uv](https://docs.astral.sh/uv/getting-started/installation/) is locally installed. Since Metamist currently is deployed with python 3.11, you should also use 3.11 locally to ensure that any changes you make are compatible with the deployed version of Metamist.
 
 
 ```bash
-# install python 3.11
-pyenv install 3.11
+# creates virtual env, install python version (3.11) and install dependencies
+uv python install 3.11
 
-# select python 3.11 for your current shell
-pyenv shell 3.11
-
-# set up virtualenv with python 3.11
-virtualenv .venv -p 3.11
+uv venv --seed
+uv sync --frozen --group dev
 
 # activate virtualenv
 source .venv/bin/activate
```

Comment:
Don't need this if following commands are prefixed with `uv run`

Reply:
Addressed in https://github.com/populationgenomics/metamist/pull/1129/commits/6ed2a45b4410e7bc288fc8304678dcd20c7abdaa